knowledge-base/httpcontext.current.user-is-not-available-in-the-data-retrieval-method-executed-by-an-objectdatasource-component.md

```diff
@@ -0,0 +1,19 @@
+---
+title: HttpContext.Current.User is not available in the data-retrieval method executed by an ObjectDataSource component
```

Comment:
The title shouldn't have only the error message. Change it to something like "Getting an error when ..." Then add the actual error in the description meta-tag with some additional details